guides/common/modules/proc_installing-postgresql.adoc

```diff
@@ -96,6 +96,15 @@ CREATE DATABASE candlepin OWNER candlepin;
 CREATE DATABASE pulpcore OWNER pulp;
 ----
 +
+. Connect to the Pulp database and create the `hstore` extension:
++
+[options="nowrap" subs="verbatim,quotes"]
+----
+postgres=# \c pulpcore
+You are now connected to database "pulpcore" as user "postgres".
+pulpcore=# CREATE EXTENSION IF NOT EXISTS "hstore";
+CREATE EXTENSION
```

Comment:
```suggestion
----
+
You are now connected to database "pulpcore" as user "postgres".

. Create an `hstore` extension:
[options="nowrap" subs="verbatim,quotes"]
----
CREATE EXTENSION
```
Can this be better @maximiliankolb @evgeni ?

Reply:
Splitting commands sounds like a good idea to me.

Reply:
updated, have a look please